src/scenes/InventoryRouter/Search.tsx

```diff
@@ -84,6 +85,10 @@ export default function Search<T extends { facilityInventories?: string }>(props
     void dispatch(requestSubLocations(filters.facilityIds ?? []));
   }, [filters.facilityIds, dispatch]);
 
+  useEffect(() => {
+    void dispatch(requestProjects(selectedOrganization.id, activeLocale || undefined));
+  }, [filters.facilityIds, dispatch, selectedOrganization.id, activeLocale]);
```

Comment:
should the facilityIds be used in the request?

Reply:
Nope, good catch.